examples/getting_started/2-Customization.ipynb

```diff
@@ -16,13 +16,13 @@
     "HoloViews elements like the `Scatter` points illustrated in the [Introduction](1-Introduction.ipynb) contain two types of information:\n",
     "\n",
     "- **Your data**, in as close to its original form as possible, so that it can be analyzed and accessed as you see fit.\n",
-    "- **Metadata specifying what your data *is***, which allows HoloViews to construct a visual representation for it.\n",
+    "- **Metadata specifying what your data *is***, which allows HoloViews to construct an appropriate visual representation for it.\n",
```

Comment:
Unbalanced ``*``s I think.

Reply:
I think it is correct:

**Metadata specifying what your data *is***

Reply:
Nevermind, missed the third closing ``*``